README.md

```diff
@@ -120,6 +130,13 @@ A default `config.json.example` file is included for reference. Copy this file t
     "enabled"                  Bool    Rotation is enabled while your configured preferred_teams game is live.
     "during_inning_breaks"     Bool    Rotation is enabled while your configured preferred_teams game is live during an inning break.
 
+"weather":                             Options for retreiving the weather
+  "apikey"                     String  An API key is requires to use the weather service. You can get one for free at [Open Weather Map](https://home.openweathermap.org/users/sign_up).
+  "zipcode"                    String  The zipcode/postcode for the location you wish to receive weather data
+  "country"                    String  The ISO 3166 country code associated with the zipcode
+  "metric_units"               Bool    Set true for celsius and meters/s. Set false for fahrenheit and miles per hour.
+  "pregame"                    Bool    Add a weather report to the pregame scroll for the game location (Not yet implemented)
```

Comment:
We should remove config options that aren't implemented. Easy enough to add back in when it actually works